app/test/server/db-service.js

```diff
@@ -2,7 +2,7 @@
 const request = require('supertest')
 const should = require('should') // eslint-disable-line
 const app = require('../../app.js')
-const hudApi = require('../../lib/middleware/hudApi')
+const localAPI = require('../../lib/middleware/localAPI')
```

Comment:
I assume that this is a temporary switch, something to be reverted on deployment. Do I have that right?

Reply:
I believe it would stay like that, in order to be able to do the unit tests without a dependency upon potentially changing data.

Reply:
Oh, of course. I had not thought that through adequately. Thanks!